cmd/vic-machine/list/list.go

```diff
@@ -161,24 +161,26 @@ func (l *List) Run(clic *cli.Context) (err error) {
 
 	var validator *validate.Validator
 	if l.Data.ComputeResourcePath == "" {
-		validator, err = validate.CreateNoDCCheck(ctx, l.Data)
+		validator, err = validate.NewValidator(ctx, l.Data)
 	} else {
 		validator, err = validate.NewValidator(ctx, l.Data)
 	}
 	if err != nil {
 		log.Errorf("List cannot continue - failed to create validator: %s", err)
 		return errors.New("list failed")
 	}
+	// If dc is not set, and multiple datacenter is available, vic-machine ls will list VCHs under all datacenters.
+	validator.AllowEmptyDC()
 
 	_, err = validator.ValidateTarget(ctx, l.Data)
 	if err != nil {
 		log.Errorf("List cannot continue - target validation failed: %s", err)
-		return err
+		return errors.New("list failed")
```

Comment:
As it happened, it should be as is, but lowercase should be used for "delete" error.